src/id3manager/formats/abc.py

```diff
@@ -0,0 +1,19 @@
+import abc
+import typing as t
+
+import mutagen.id3 as id3
+
+
+class MetadataFormatter(abc.ABC):
+    @staticmethod
+    @abc.abstractmethod
+    def name() -> str:
```

Comment:
Is there any reason why you chose `@staticmethod` and not `@property` here? :thinking: If not, I think it would be nicer to go with `@property`. This way developers are forced to implement the property in their classes, but can use class properties for that.

Reply:
Oh, I wanted to make sure `abc` forces sub-classes to implement `name()`, and I somehow mistakenly thought there was no way to do that for a `@property`.